### PR TITLE
BaseExtendedPicker: Hook up onPaste

### PR DESCRIPTION
**Pull request by amyngu**
#### Pull request checklist
- [x] Include a change request file using `$ npm run change`

#### Description of changes
- Hook up onPaste
- Hook up onItemSelected
- Expose input element
- Add generic classname for EditingItem

### CODE REVIEW

File: packages/experiments/src/components/ExtendedPicker/BaseExtendedPicker.tsx [hunk 1]
```diff
@@ -66,9 +66,14 @@ export class BaseExtendedPicker<T, P extends IBaseExtendedPickerProps<T>> extend
   }
 
   public componentDidMount(): void {
+    this.input.inputElement.addEventListener('paste', this.onPaste);
     this.forceUpdate();
   }
 
+  public componentWillUnmount(): void {
+    this.input.inputElement.removeEventListener('paste', this.onPaste);
```
Comment (dzearing) [inline packages/experiments/src/components/ExtendedPicker/BaseExtendedPicker.tsx]: this seems like a bad idea. Why are we not using react eventing?

**Comment by amyngu** [comment]
@dzearing Ping. Any other comments? Thanks!

**Comment by amyngu** [comment]
@dzearing Ping. Any other comments? Thanks!

**Comment by amyngu** [comment]
@dzearing I think this is a simple change and I'd really appreciate a review/sign off!

File: packages/experiments/src/components/SelectedItemsList/SelectedPeopleList/Items/ExtendedSelectedItem.tsx [hunk 2]
```diff
@@ -88,8 +88,10 @@ export class ExtendedSelectedItem extends BaseComponent<ISelectedPeopleItemProps
       </div >);
   }
 
-  private onClickIconButton = (action: (() => void) | undefined): () => void => {
-    return (): void => {
+  private onClickIconButton = (action: (() => void) | undefined): (ev: React.MouseEvent<HTMLAnchorElement | HTMLButtonElement>) => void => {
```
Comment (joschect) [inline packages/experiments/src/components/SelectedItemsList/SelectedPeopleList/Items/ExtendedSelectedItem.tsx]: Why not just make this a method? Also private names should start with _

Comment (amyngu) [reply packages/experiments/src/components/SelectedItemsList/SelectedPeopleList/Items/ExtendedSelectedItem.tsx]: I'll change the function name. 

I'm not sure I follow the first part of your comment. Do you mean make separate methods for the onRemoveItem and onExpandItem?

Comment (amyngu) [reply packages/experiments/src/components/SelectedItemsList/SelectedPeopleList/Items/ExtendedSelectedItem.tsx]: Right, I'll change the function name.

I'm not sure I follow the first part of your comment. Do you mean make separate methods for the onRemoveItem and onExpandItem?

**Pull request merged**